CHANGELOG.md

```diff
@@ -0,0 +1,135 @@
+# Changelog
```

Comment:
That's a great idea, I missed something like this. I also very much like split on separate sections, with different target groups.